README.md

```diff
@@ -19,9 +19,23 @@ $ enry
 
 Note that enry's CLI **_does not need an actual git repository to work_**, which is an intentional difference from linguist.
 
+## Usage
 
-Install
--------
+By default, only the languages from Markdown and Programming groups are reported.
+
+One can choose to report all the languages (including plain text, markup, templates, configuration scripts, etc) using
+```
+enry -all
+```
+
+For cases when subsequent lexing/parsing of the files is desired, one can keep only the programming languages by
+```
+enry -prog
```

Comment:
instead of just adding this specific case you can add --type=[prose|programming|etc...]

Reply:
Nice suggestion https://github.com/go-enry/enry/issues/7

Reply:
this change then will be a breaking change, so it's not very nice to introduce this flag and in the future replace and remove/deprecate this.

Reply:
> this change then will be a breaking change, so it's not very nice to introduce this flag and in the future replace and remove/deprecate this.

sorry, somehow I missed that! You are right, #7 updated to reflect this.